src/openforms/js/lang/formio/en.json

```
@@ -35,6 +35,7 @@
   "maxFileSizeMessage": "The maximum file size is {{maxFileSize}}.",
   "removeFileMessage": "Remove {{fileName}}",
   "The uploaded file is not of an allowed type. It must be: {{ pattern }}.": "The uploaded file is not of an allowed type. It must be: {{ pattern }}.",
+  "uploadFile": "upload {{fileFieldLabel}}",
```

sort of discussed in the office - I think this should be modified to be:

```suggestion
  "uploadFile": "selecteer een \"{{ fileFieldLabel }}\"-bestand",
```

Additionally, we should only use this label via a `aria-label` helper on the link rather than changing the UI for non-visually impaired users too, that should resolve the items from the a11y report.